docs/style.md

```diff
@@ -0,0 +1,140 @@
+<!-- alex disable hooks -->
+
+<!--
+Inspired by Uber's Go Style Guide:
+https://github.com/uber-go/guide/blob/85bf203f4371a8ae9e5e9a4d52ea77b17ca04ae6/style.md
```

Comment:
just wondering is there something specific about this commit vs linking to their master branch?

Reply:
@emyarod just wanted a permalink so if the file in `master` moves paths the link won't be dead 👍